src/org/usfirst/frc/team2706/robot/subsystems/Bling.java

```diff
@@ -0,0 +1,153 @@
+    *other signals to display the critical battery warning
+    */
+
+    //Let's make the colour and command codes
+    Map<String, String> colours = new HashMap<String, String>(){{
+    put("RED", "16711680");
+    put("GREEN", "32768");
+    put("YELLOW", "65535");
+    put("PURPLE", "8388736");
+    put("ORANGE", "16753920");
+    put("BLUE", "255");
+    put("VIOLET", "15631086");
+    put("MERGE", "5374156");
+    put("TAN", "16767411");
+    put("PINK", "14027935");
+    put("WHITE", "16777215");
+    put("TURQUOISE", "65535");
+    put("BLACK", "0");
+    }};
+    
+	public Bling() {
+	    
+	    blingPort.setTimeout(0.5); //Will wait a max of half a second.
+	    blingPort.writeString("I");
+		String readString = blingPort.readString(); //Get output from the arduino.
+		
+	}
+	
+	/**
+	 * This function should be run at the beginning of autonomous to get
+	 * the proper light pattern.
+	 */
+	public void auto() { //Will run during autonomous period
+	    
+		blingPort.writeString("I");
+		blingPort.writeString("E0Z"); //Clear the LED strip
+		blingPort.writeString("E1Z");
+	}
+	
+	/**
+	 * This function runs to initialize the teleop patterns.
+	 * Starts the teleop period for the bling
+	 */
+	public void startTelOp() {
+	    
+		blingPort.writeString("I");
+		blingPort.writeString("E0Z"); //Clear the LED strip
+	}
+	
+	/**
+	 * This function will run whenever we want to display the
+	 * battery voltage output. Will run automatically at startup.
+	 * @param percent : The current voltage percent reading from the battery.
+	 */
+	public void batteryInd(double percent) { 
+	    
+	    blingPort.writeString("I"); //Let them know we need to send another command
+	    blingPort.writeString("E0Z"); //Clear the LED strip
+	    String bColour;
+	    if (percent <= 0.25) bColour = colours.get("RED");
+	    else if (percent <= 0.5) bColour = colours.get("YELLOW");
+	    else if (percent <= 0.75) bColour = colours.get("PURPLE");
+	    else bColour = colours.get("GREEN");
+	    
+	    //Use multi-colour display
+	    blingPort.writeString("F12C" + bColour + "D" + Math.round(percent * 100) + "E12Z"); 
+	}
+	/**
+	 * Show a distance indication on the LED strip out of 3 metres.
+	 * @param distance
+	 * : The current distance
+	 */
+	public void showDistance(double distance) {
+	    
+	    blingPort.writeString("I");
+	    blingPort.writeString("E0Z"); //Clear the LED strip
+	    if (distance > 3.0) return; //Only showing 3 metres from the object.
+	    double percentDist = distance/3;
+	    System.out.println(Math.round(percentDist * pixels));
+	    String dColour;
+	    if (distance > 2) dColour = colours.get("RED");
+	    else if (distance > 1) dColour = colours.get("YELLOW");
+	    else dColour = colours.get("GREEN");
+	    System.out.println(dColour);
+	    blingPort.writeString("F7C" + dColour + "P0" + "Q" + Math.round(percentDist * pixels) + "E7Z"); //Colour flash with different colours
+	}
```

Comment:
The functions above had the } after a space, either choose closing the function right below the line or add a new line, personally I like closing it without a new line like this one :)